Core/Object Arts/Dolphin/ActiveX/OCX/AXControlSite.cls

```diff
@@ -795,6 +795,29 @@ state
 	sequence list addAll: controlState afterIndex: 1.
 	^sequence!
 
+storeOn: aStream indent: anInteger variables: aDictionary
+
+	| default name |
+	name := self storeNameIn: aDictionary.
+	aStream display: name; cr.
+	anInteger + 1 timesRepeat: [aStream tab].
+	[
+		default := (self class progId: self progId) create; show.
```

Comment:
The goal was to write code that looked as human-written as possible by leaving out the values that would otherwise be set by the initialize process. To the extent that it does create new instances, it is in the context of View, and the view composer does the same. On the other hand, if you'd prefer I can set all the values explicitly, even if they are the same as the default values; it is just cluttering the code with things that a human author would leave out.